spec/identity_doc_auth/lexis_nexis/error_generator_spec.rb

```diff
@@ -0,0 +1,233 @@
+      PortraitMatchResults: { FaceMatchResult: pm_result },
+      ImageMetrics: {},
+    }
+  end
+
+  context 'The correct errors are delivered with liveness off when' do
+    it 'DocAuthResult is Attention' do
+      error_info = build_error_info(
+        'Attention',
+        build_alerts({name: '2D Barcode Read', result: 'Attention'}),
+        nil)
+
+      output = described_class.new(config).generate_trueid_errors(error_info, false)
+
+      expect(output.keys).to contain_exactly(:back)
+      expect(output[:back]).to contain_exactly(
+        i18n.t('doc_auth.errors.lexis_nexis.barcode_read_check')
+      )
+    end
+
+    it 'DocAuthResult is Failed' do
+      error_info = build_error_info(
+        'Failed',
+        build_alerts({name: 'Visible Pattern', result: 'Failed'}),
+        nil)
+
+      output = described_class.new(config).generate_trueid_errors(error_info, false)
+
+      expect(output.keys).to contain_exactly(:id)
+      expect(output[:id]).to contain_exactly(
+        i18n.t('doc_auth.errors.lexis_nexis.id_not_verified')
+      )
+    end
+
+    it 'DocAuthResult is Failed with multiple different alerts' do
+      error_info = build_error_info(
+        'Failed',
+        build_alerts({name: '2D Barcode Read', result: 'Attention'},
+                     {name: 'Visible Pattern', result: 'Failed'}),
+        nil)
+
+      output = described_class.new(config).generate_trueid_errors(error_info, false)
+
+      expect(output.keys).to contain_exactly(:general)
+      expect(output[:general]).to contain_exactly(
+        i18n.t('doc_auth.errors.lexis_nexis.general_error_no_liveness')
+      )
+    end
+
+    it 'DocAuthResult is Failed with multiple id alerts' do
+      error_info = build_error_info(
+        'Failed',
+        build_alerts({name: 'Expiration Date Valid', result: 'Attention'},
+                     {name: 'Full Name Crosscheck', result: 'Failed'}),
+        nil)
+
+      output = described_class.new(config).generate_trueid_errors(error_info, false)
+
+      expect(output.keys).to contain_exactly(:id)
+      expect(output[:id]).to contain_exactly(
+        i18n.t('doc_auth.errors.lexis_nexis.general_error_no_liveness')
+      )
+    end
+
+    it 'DocAuthResult is Failed with multiple back alerts' do
+      error_info = build_error_info(
+        'Failed',
+        build_alerts({name: '2D Barcode Read', result: 'Attention'},
+                     {name: '2D Barcode Content', result: 'Failed'}),
+        nil)
+
+      output = described_class.new(config).generate_trueid_errors(error_info, false)
+
+      expect(output.keys).to contain_exactly(:back)
+      expect(output[:back]).to contain_exactly(
+        i18n.t('doc_auth.errors.lexis_nexis.multiple_back_id_failures')
+      )
+    end
+
+    it 'DocAuthResult is Failed with an unknown alert' do
+      error_info = build_error_info(
+        'Failed',
+        build_alerts({name: 'Not a known alert', result: 'Failed'}),
+        nil)
```

Comment:
ok so I probably should have copied over some sort of rubocop config, but this one is bugging me --- here the closing paren is on the same line, but down on lines 138-140, the paren goes to the new line? Can we make these consistent with each other?

Reply:
Yep.